### PR TITLE
Add TIC1 and TIC5 for ice restarts and fix divide by zero with AVG

### PR DESCRIPTION
**Pull request by dabail10**
# Pull Request Summary
 

This adds some changes from EMC/develop for sea ice related restarts and also a divide by zero.

### Issue(s) addressed

Already added to EMC/develop.

### Check list  

- [ ] Branch is up to date with the authoritative repository (NOAA-EMC) develop branch. 
- [ ] Checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop). 
- [ ] If a version number update is required, checked the [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number) checklist. 
- [ ] If a new feature was added, a regression test for testing the new feature is added. 
 
### Testing

* How were these changes tested?
* Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)
* Have the matrix regression tests been run (if yes, please note HPC and compiler)?
* Please indicate the expected changes in the regression test output, (Note the [list](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-develop#4-look-at-results) of known non-identical tests.)
* Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):

### CODE REVIEW

File: model/src/w3iorsmd.F90 [hunk 1]
```diff
@@ -305,8 +305,7 @@ SUBROUTINE W3IORS ( INXOUT, NDSR, DUMFPI, IMOD, FLRSTRT )
     USE W3GDATMD, ONLY: NX, NY, NSEA, NSEAL, NSPEC, MAPSTA, MAPST2, &
          GNAME, FILEXT, GTYPE, UNGTYPE
     USE W3TRIAMD, ONLY: SET_UG_IOBP
-    USE W3WDATMD, only : DINIT, VA, TIME, TLEV, TICE, TRHO, ICE, UST
-    USE W3WDATMD, only : USTDIR, ASF, FPIS, ICEF, TIC1, TIC5, WLV
+    USE W3WDATMD
```
Comment (alperaltuntas) [inline model/src/w3iorsmd.F90:308]: Why is this change needed?

**Comment by dabail10** [comment]
Not necessary, but this was the change in EMC/develop.

File: model/src/w3fld1md.F90 [hunk 2]
```diff
@@ -1116,7 +1116,11 @@ SUBROUTINE APPENDTAIL(INSPC, WN2, NKT, KA1, KA2, KA3, WNDDIR,SAT)
     DO K=KA1, KA2-1
       AVG=SUM(INSPC(K,:))/MAX(REAL(NTH),1.)
       DO T=1,NTH
-        INSPC(K,T)=BT(K)*INSPC(K,T)/TPI/(WN2(K)**3.0)/AVG
+        if (avg /= 0) then
+          INSPC(K,T)=BT(K)*INSPC(K,T)/TPI/(WN2(K)**3.0)/AVG
+        else
+          inspc((k,t) = 0.0
```
Comment (alperaltuntas) [inline model/src/w3fld1md.F90]: typo here: inspc((k,t)

**Review by alperaltuntas — CHANGES_REQUESTED** [review]

**Comment by dabail10** [comment]
Whoops.
